Gemfile

```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'annotate', '~> 3.2', '>= 3.0.3'
   gem 'bullet', '~> 7.0.3'
   gem 'byebug'
+  gem 'capybara'
```

Comment:
We should add version to prevent undesired updates

Reply:
Ok!